pkg/oc/clusteradd/components/registry/registry_install.go

```diff
@@ -1,102 +0,0 @@
-package registry
```

Comment:
This could be left as is and turned into optional component if the admin wishes to spawn legacy docker-registry.

Reply:
i'm all for deleting it. :)